src/main/scala/com/github/simplesteph/ksm/compat/AdminClientAuthorizer.scala

```diff
@@ -3,7 +3,7 @@ package com.github.simplesteph.ksm.compat
 import java.util
 
 import kafka.network.RequestChannel
-import kafka.security.SecurityUtils
+import kafka.security.authorizer.AuthorizerWrapper
```

Comment:
so the problem with this is that this class is deprecated, so I'm not a fan of using it. It works as a quick fix though

https://github.com/apache/kafka/blob/90bbeedf52f4b6a411e9630dd132583afa4cd428/core/src/main/scala/kafka/security/authorizer/AuthorizerWrapper.scala#L41

Reply:
damn, was hoping it would be a quick fix. if i get some more time i'll try and look into `kafka.security.authorizer.AclAuthorizer`

Reply:
this can be in two steps. First, tell me if that fixes your issue with `Allow`. The code should still build and run. Then we can see the next steps

Reply:
Yep, i confirmed before creating this PR that this works with our cluster running`2.4.1`